spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/SchedulerService.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.server.service;
+
+
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.cloud.scheduler.spi.core.ScheduleInfo;
+import org.springframework.cloud.scheduler.spi.core.ScheduleRequest;
+import org.springframework.data.domain.Pageable;
+
+/**
+ * Provides Scheduler related services.
+ *
+ * @author Glenn Renfro
+ */
+public interface SchedulerService {
+
+	/**
+	 * Registers the {@link ScheduleRequest} to be executed based on the
```

Comment:
Could you elaborate more on what is getting executed here? I am not sure where does the cron expression come from.

Reply:
Good point.   I'll update the documentation.